admin/src/main/resources/io/buoyant/admin/template/metric.partial.template

```diff
@@ -0,0 +1,16 @@
+<div class="{{containerClass}}" style={{#if borderColor}}"border-left: 1px solid {{borderColor}}"{{/if}}>
+  <div class="metric-header">
+    {{#if description}}
+      {{description}}
+    {{else}}
+      metric
+    {{/if}}
```

Comment:
I added an `update()` before we call `setInterval` so now, when we load the page, we shouldn't see blank spots in the client section.  We still see it when we add a new client, as we need to wait for the next call to `update()`... so I'll leave "metric" in as a placeholder for this case.